spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,14 @@
+feature 'User sign in' do
+
+ scenario 'with correct credentials' do
+   user = create(:user)
+   sign_in(user)
+   expect(page).to have_content "Welcome, #{user.full_name}"
+ end
+
+  scenario 'with incorrect credentials' do
+    user = build(:user, username: "wrong")
+    sign_in(user)
+    expect(page).to have_content "Sorry, there were the following problems with the form."
+    end
```

Comment:
`end` at 13, 4 is not aligned with `scenario 'with incorrect credentials' do` at 9, 2